digitalpy/core/zmanager/action_mapper.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC
+from digitalpy.core.zmanager.request import Request
+
+from digitalpy.core.zmanager.response import Response
+
+
+class ActionMapper(ABC):
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>B024 ActionMapper is an abstract base class, but it has no abstract methods. Remember to use the @abstractmethod decorator, potentially in conjunction with @classmethod, @property and/or @staticmethod.